### PR TITLE
Update Home.tsx

### PR DESCRIPTION
**Pull request by TechnicallyGio**
Uses Modal to create flights rather than changing the view focus. Modal needs to be wider.

### CODE REVIEW

File: resources/pages/Home/Home.tsx [hunk 1]
```diff
@@ -55,17 +55,53 @@ export const Home = (): JSX.Element => {
...
-          if (flightsCardRef.current !== null) {
-            return flightsCardRef.current.scrollIntoView();
-          }
-          return itineraryCardRef.current?.scrollIntoView();
-        }}
+        onGetStarted={() => setIsItineraryModalOpen(true)}
+
       />
+
+      <Modal
```
Comment (SuperCheese21) [inline resources/pages/Home/Home.tsx:67]: You can use the `className` prop on the `Modal` component to set the width manuallu

File: resources/pages/Home/Home.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { zodResolver } from '@hookform/resolvers/zod';
 import { useEffect, useRef, useState } from 'react';
 import { useForm } from 'react-hook-form';
 import { useNavigate } from 'react-router-dom';
-import { Form } from 'stratosphere-ui';
+import {Form, Modal} from 'stratosphere-ui';
```
Comment (SuperCheese21) [inline resources/pages/Home/Home.tsx:5]: Looks like you don't have format on save enabled. Could you please run `npm run format` and commit the changes?

File: resources/pages/Home/Home.tsx [hunk 3]
```diff
@@ -55,17 +55,53 @@ export const Home = (): JSX.Element => {
...
+                  setFlights([]);
+                  setIsResetDialogOpen(false);
+                  firstFieldRef.current?.focus();
+                }}
+                open={isResetDialogOpen}
+            />
+        ) : null}
+      </Modal>
+
+      {/*
```
Comment (SuperCheese21) [inline resources/pages/Home/Home.tsx:104]: Please remove this code if it is not needed anymore

File: resources/pages/Home/Home.tsx (hunk 1 above)
Comment (TechnicallyGio) [reply resources/pages/Home/Home.tsx:67]: Did not work

File: resources/pages/Home/Home.tsx (hunk 2 above)
Comment (TechnicallyGio) [reply resources/pages/Home/Home.tsx:5]: will do

File: resources/pages/Home/Home.tsx (hunk 3 above)
Comment (TechnicallyGio) [reply resources/pages/Home/Home.tsx]: Resolved

**Comment by SuperCheese21** [comment]
Form is not responsive. Please make sure modal width adjusts in mobile mode.

**Review by SuperCheese21 — CHANGES_REQUESTED** [review]
- [x] Form is not responsive. Please make sure modal width adjusts in mobile mode.

**Review by SuperCheese21 — APPROVED** [review]

**Pull request merged**